src/services/CustomerDeliveryGenerator.ts

```
@@ -388,6 +402,8 @@ export class CustomerDeliveryGenerator {
         priority: {
           $switch: {
             branches: [
+              { case: { $lte: ['$daysSinceLastDelivery', 2] }, then: 'LOW' },
+              { case: { $eq: ['$isDeliveryFrequencyUrgent', true] }, then: 'URGENT' },
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)